File: sample/src/main/java/contacts/sample/cheatsheet/other/java/ContactShareActivity.java
package contacts.sample.cheatsheet.other.java;

import android.app.Activity;
import android.content.Intent;

import java.util.List;

import contacts.core.entities.Contact;
import contacts.core.util.ContactShareKt;

public class ContactShareActivity extends Activity {

    void shareContact(Contact contact) {
        Intent shareIntent = ContactShareKt.shareVCardIntent(contact);
        if (shareIntent != null) {
            startActivity(Intent.createChooser(shareIntent, null));
        }
    }

    void shareContacts(List<Contact> contacts) {
        Intent shareIntent = ContactShareKt.shareMultiVCardIntent(contacts);
        if (shareIntent != null) {
            startActivity(Intent.createChooser(shareIntent, null));
        }
    }
}

File: sample/src/main/java/contacts/sample/cheatsheet/other/kotlin/ContactShareActivity.kt
package contacts.sample.cheatsheet.other.kotlin

import android.app.Activity
import android.content.Intent
import contacts.core.entities.Contact
import contacts.core.util.*

class ContactShareActivity : Activity() {

    fun shareContact(contact: Contact) {
        val shareIntent = contact.shareVCardIntent()
        if (shareIntent != null) {
            startActivity(Intent.createChooser(shareIntent, null))
        }
    }

    fun shareContacts(contacts: List<Contact>) {
        val shareIntent = contacts.shareMultiVCardIntent()
        if (shareIntent != null) {
            startActivity(Intent.createChooser(shareIntent, null))
        }
    }
}